docs/server/source/dev-and-test/setup-bdb-host.md

```diff
@@ -0,0 +1,58 @@
+#### Set up BigchainDB node on Local Dev Machine
+
+### With RethinkDB
```

Comment:
Let's reorder these two subsections so that MongoDB is first and RethinkDB is second. We're trying to encourage the use of MongoDB rather than RethinkDB.

Reply:
+1